core-bundle/src/Resources/contao/library/Contao/Database/Result.php

```diff
@@ -124,7 +124,8 @@ public function __isset($strKey)
 			$this->next();
 		}
 
-		return isset($this->resultSet[$this->intIndex][$strKey]) || isset($this->arrModified[$strKey]);
+		// Use array_key_exists() instead of isset(), because the value might be null
+		return \array_key_exists($strKey, $this->resultSet[$this->intIndex]) || \array_key_exists($strKey, $this->arrModified);
```

Comment:
```suggestion
		return \array_key_exists($strKey, $this->arrModified) ? isset($this->arrModified[$strKey]) : isset($this->resultSet[$this->intIndex][$strKey]);
```

Reply:
Why do we need the extra `array_key_exists()` in your suggestion? Isn't the result the same as with the original code?

Reply:
If the value in `$this->arrModified` was set to `null` `__isset` should return `false` event if the value exists in `$this->resultSet` I think.

Reply:
Agreed. 👍